tests/test_pyrflx.py

```diff
@@ -1558,8 +1568,17 @@ def test_checksum_manual_icmp(icmp_checksum: MessageValue) -> None:
     icmp_checksum.set("Identifier", 5)
     icmp_checksum.set("Sequence_Number", 1)
     icmp_checksum.set("Data", test_data)
+    assert not icmp_checksum.valid_message
     assert icmp_checksum.get("Checksum") == 1234
-    assert icmp_checksum.bytestring == b"\x08\x00\x04\xd2\x00\x05\x00\x01" + test_data
+    with pytest.raises(RuntimeError, match="cannot create bytestring of invalid message"):
+        assert icmp_checksum.bytestring == b"\x08\x00\x04\xd2\x00\x05\x00\x01" + test_data
```

Comment:
`assert_bytestring_error` should be also used here.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/431/commits/8c5a6ffb899ee8f75baeec7f7380bd27ba434f93.